dependencies.gradle

```diff
@@ -1,28 +1,28 @@
 ext {
     versions = [
-            support            : '25.3.1',
-            androidSdk         : [
+            support   : "25.3.1",
```

Comment:
I think that the convention in Groovy is to use single quotes where you don't do string interpolation 😛